site/source/docs/tools_reference/emcc.rst

```diff
@@ -282,6 +282,9 @@ Options that are modified or new in *emcc* are listed below:
 ``--exclude-file <name>``
 	Files and directories to be excluded from :ref:`--embed-file <emcc-embed-file>` and :ref:`--preload-file <emcc-preload-file>`. Wildcards (*) are supported.
 
+``--fully-prepare-files``
```

Comment:
Hmm, the word `extra` sounds equally superlative, like there is a "no prepare", "standard prepare" and then the all-inclusive "extra-prepare"? Doesn't this file packager/emcc option control tell the browser to decode the assets to audio and image elements, in order to get past the async->sync hiccup? (If we did have access to synchronous decoding, this feature would probably not have existed in the first place(?))